tshirts.c

```diff
@@ -17,6 +17,7 @@ int main() {
     assert(size(37) == 'S');
     assert(size(40) == 'M');
     assert(size(43) == 'L');
+    assert((size(38) == 'S')||(size(38) == 'M')||(size(42) == 'M')||(size(42) == 'L'));

```

Comment:
Do you expect size 38 to be S or M?
you need to clearly state the expected behavior (whatever you assume) using asserts and cover all range of values with boundary values.